src/app/components/form/form.component.ts

```diff
@@ -30,9 +34,10 @@ export class FormComponent implements OnInit {
             plaats: '',
             afstand: '',
             sleutelwoorden: ''
-        });
+        })
 
     submit = () => {
```

Comment:
```suggestion
    submit: void {
```

Also change for getForm

Reply:
Changed. For your information:
```typescript
submit: void {
```
will not work, but
```typescript
submit(): void {
```
will work